share/wake/lib/system/job.wake

```diff
@@ -337,7 +337,7 @@ def getPath input =
 
 # wakeroot is the absolute sandbox-path from which input and output files will
 # be interpreted as being relative to if they're in fact relative.
-export def mkJobCacheRunner (wakeroot: String) ((Runner name score baseDoIt): Runner): Runner =
+export def mkJobCacheRunner (hashFn: Result RunnerInput Error => Result String Error) (wakeroot: String) ((Runner name score baseDoIt): Runner): Runner =
```

Comment:
Could this be a `String`? The `RunnerInput` is already inspect-able by the cache runner

Reply:
What specifically do you want to be a string?

Reply:
Like this:
```suggestion
export def mkJobCacheRunner (hash: String) (wakeroot: String) ((Runner name score baseDoIt): Runner): Runner =
```

Reply:
You can do that with a function that always returns the same string instead. This is a more generic interface.

Reply:
ok